setup/setup.sh

```diff
@@ -2,16 +2,15 @@
 #
 # Pre-reqs:
 # 1. Linux machine: Debian/Ubuntu/...
-# 2. Files from our setup folder locally in a local folder
+# 2. setup.sh file from our setup folder locally in a local folder
 #    (pulled from Github or otherwise).
-#    *MUST HAVE: docker-compose.yml* and MUST update sercrets below.
-# 3. Update up.sh with real passwords before the first run!
 #
 # Give any arg to skip the installation, e.g. "./setup.sh local"
 
-# Create .env file with secrets (***UPDATE BEFORE RUNNING!***):
+# Create necessary files.
+
 cat >.env << ENV
-# Setting secrets, update before running.
+# Setting secrets, please, update with real passwords and save/exit editor.
```

Comment:
This comment isn't fully clear. I'm seeing the vim .env which looks correct. This comment and variables definition in this file doesn't look relevant to it.

Reply:
Expanded. This comment does not belong to setup.sh, it's the content of generated .env file. This way this is at least clear, other ways to insert such instruction into .env file will be much more confusing.